zebrad/Cargo.toml

```diff
@@ -168,6 +168,7 @@ toml = "0.8.3"
 futures = "0.3.29"
 rayon = "1.7.0"
 tokio = { version = "1.33.0", features = ["time", "rt-multi-thread", "macros", "tracing", "signal"] }
+tokio-stream = { version = "0.1.14", features = ["time"] }
```

Comment:
I'd like a bit more time to finish reviewing this PR before it gets merged, so I'm opening this dummy comment. I'll resolve it manually soon.

Reply:
We won't be doing anything with the release until after tomorrow's meeting about PR #7817, and after the [full sync](https://github.com/ZcashFoundation/zebra/actions/runs/6686504232) finishes.

So you have plenty of time.